styles/main.css

```diff
@@ -162,7 +162,6 @@ nav a:hover{
 }
 
 .back-to-top a{
-    position: static;
+    position: fixed;
     bottom: 3.5em;
-    right: 3.5em;

```

Comment:
You actually don't want this, as this will put the link 3.5 from the right of the browser window, but you want it to be at the right of the `.back-to-top` div, which is what the text-align rule did for you.  Leaving `right` unspecified just leaves the element in its normal flow position with respect to the right value